docs/source/how-tos/create_enrollment_requests_manually.rst

```diff
@@ -1,2 +1,42 @@
 Create enrollment requests manually
 ====================================
+
+The Open edX WooCommerce Plugin uses Enrollment Request to manage the enrollments generated by the WooCommerce integration, but you can create an Enrollment Request manually.
+
+In this section, you will learn how to create an Enrollment Request manually.
+
+Requisites
+-----------
+
+To connect with the Open edX platform, you must have valid settings for this plugin. If you don't have this plugin set yet, the how-to :doc:`Create an Open edX Application for the Plugin Settings </how-tos/create_an_openedx_app>` will be helpful.
+
+Steps
+------
+
+#. Enter the Enrollments Manager option in your WordPress dashboard's Open edX Sync tab.
+
+    .. image:: /_images/how-tos/create_enroll_request/menu.png
+        :alt: Enrollments Manager option
+
+#. Create a new Enrollment Request (URL: `<domain>/wp-admin/post-new.php?post_type=openedx_enrollment`)
+
+    .. image:: /_images/how-tos/create_enroll_request/new_enroll_request.png
+        :alt: New Enrollment Request
+
+* Use a valid course ID that will be used in the request. e.g. course-v1:edX+DemoX+Demo_Course.
+
+* Enter the email of the user who will be used in the request.
+
+* Select a Course Mode that the course has.
+
+* Select the type of request you want to execute. If you select Enroll, you will create an enrollment, and if you select Un-enroll, you will set a soft unenrollment (enrollment with status inactive).
```

Comment:
Enroll / Unenroll should be in bold, more noticable.

Reply:
Thanks Wanda :D